app/gui2/src/components/ExecutionModeSelector.vue

```diff
@@ -112,4 +115,17 @@ span {
     background: var(--color-dim);
   }
 }
+
+.dropdown-enter-active,
+.dropdown-leave-active {
+  transition: all 0.1s;
```

Comment:
```suggestion
  transition: all 0.1s;
  
  > span {
    transition: all 1s;
  }
```

along with
```css
  display: flex;
  flex-flow: column;
```
on `.execution-mode-dropdown` itself

Reply:
(this change, along with a few others, is for the transition to go smoothly from 0 height to full height)